src/modules/todo.js

```diff
@@ -11,8 +11,8 @@ class TODO {
 
   addTodo() {
     this.todos = this.readTodo();
-    this.description = myInput.value;
-    this.index = this.todos.length + 1;
+    this.description = descInput.value;
+    this.index = this.todos.length;
     this.todos.push({
       index: this.index,
```

Comment:
- [ ] **Optional:** The index key of your object should begin from 1 rather than 0.

**Hint** `index: this.index++`